chart/cas-cif/templates/networkPolicies/metabase-ingress.yaml

```diff
@@ -0,0 +1,18 @@
+kind: NetworkPolicy
+apiVersion: networking.k8s.io/v1
+metadata:
+  name: {{ .Release.Name }}-metabase-access
+  labels: {{ include "cas-cif.labels" . | nindent 4 }}
+spec:
+  podSelector:
+    matchLabels:
+      release: {{ .Release.Name }}
+  ingress:
+  - from:
+    - namespaceSelector:
+        matchLabels:
+          name: {{ .Values.ggircs.prefix }}
+          environment: {{ .Values.ggircs.environment }}
```

Comment:
You can match the `kubernetes.io/metadata.name` label, so that you don't need to break down the namespace into prefix and environment, and don't need to add more values to the chart (you can revert changes to the Makefile)